examples/nestjs/consumer/test/pact/publish.ts

```diff
@@ -0,0 +1,30 @@
+import { NestFactory } from '@nestjs/core';
+import { Publisher } from '@pact-foundation/pact';
+import { PactModuleProviders } from 'nestjs-pact';
+import { PactModule } from './pact.module';
+
+(async () => {
+  const app = await NestFactory.createApplicationContext(PactModule);
+
+  const publisher: Publisher = app.get(PactModuleProviders.PactPublisher);
+  const a = 1;
+  // const logger: LoggerService = app.get(Logger);
+
+  // if (process.env.CI !== 'true') {
+  //   console.log('Skipping Pact publish as not on CI');
```

Comment:
This is related to the comment on package json- probably you don't want pact publishing to run when you're not on CI.

Reply:
Resolved in the other branch